SETUP-pyenv.md

```diff
@@ -1,48 +1,47 @@
-## Setup Aotearoa Open Fisca in pyenv
+## Setup Aotearoa OpenFisca
 
-### Step 1: Install pyenv
-
-```sh
-curl https://pyenv.run | bash
-```
-
-### Step 2: Clone the repo
+### Step 1: Clone the repo
 
 ```sh
 git clone https://github.com/ServiceInnovationLab/openfisca-aotearoa.git
 cd openfisca-aotearoa
 ```
 
-### Step 3: Install python using pyenv
-We want to use the exact version of python we use on the official servers
+### Step 2: Install Python
+
+Install [pyenv](https://github.com/pyenv/pyenv) and [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv).
+
+And then add to `.bashrc` or `.zshrc`:
 
 ```sh
- pyenv install < .python-version
- python --version # This should match the version in .python-version file
+eval "$(pyenv init -)"
+if which pyenv-virtualenv-init >/dev/null; then eval "$(pyenv virtualenv-init -)"; fi
 ```
 
-### If `ModuleNotFoundError: No module named '_ctypes'`
->
->     sudo apt-get update
->     sudo apt-get install libffi-dev
->
-More problems? [Try the instructions here](https://stackoverflow.com/questions/27022373/python3-importerror-no-module-named-ctypes-when-using-value-from-module-mul#41310760)
+And then run:
 
+```sh
+pyenv install 3.10.4
+pyenv virtualenv openfisca-aotearoa-3.10.4
```

Comment:
at this step i got:
```
brenda@kotare:~/projects/openfisca-aotearoa$ pyenv virtualenv openfisca-aotearoa-3.10.4
pyenv: version `openfisca-aotearoa-3.10.4' is not installed (set by /home/brenda/projects/openfisca-aotearoa/.python-version)
```

I edited `.python-version` to just say `3.10.4` and tried again. This time it worked.